client/templates/minutes/minutesEditParticipants.html

```diff
@@ -35,7 +51,7 @@
                         </div>
                         {{/each}}
                     </div>
-
+                </div>
```

Comment:
It seems that this closing div-tag is at the wrong place. The following free-text field as well as the list of informed users should be included within this div. So this closing-div tag should be moved down to line 49 as it was located before. This might also solve the issue reported by @derwok as a global comment to this PR.